src/Impostor.Server/Events/Game/GameHostChangeEvent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Impostor.Api.Events;
+using Impostor.Api.Games;
+using Impostor.Api.Net;
+
+namespace Impostor.Server.Events
+{
+    public class GameHostChangeEvent : IGameHostChangeEvent
+    {
+        public GameHostChangeEvent(IGame game, IClientPlayer host)
+        {
+            Game = game;
+            Host = host;
+        }
+
+        public IGame Game { get; }
+
+        public IClientPlayer Host { get; }
```

Comment:
At the time when you get the event you can't get the old host anymore, so add a property for previous/old host?